rollup.types.config.ts

```diff
@@ -25,6 +25,33 @@ export default rollup.defineConfig({
     return externals.some(external => source.startsWith(external))
   },
   plugins: [
+    {
+      // Remove "declare module 'vscode'" from vscode type
+      // Or else dts plugin is unable to extract types from it
+      name: 'remove-vscode-declare-module',
+      transform: (code, id) => {
+        if (id.endsWith('@types/vscode/index.d.ts')) {
+          const lines = code.split('\n')
+          let index = lines.indexOf('declare module \'vscode\' {')
+          lines.splice(index, 1)
+          if (index >= 0) {
+            let end = false
+            while (!end) {
```

Comment:
Can be rewritten using a for loop?
```
for (; lines[index] !== '}'; index++) {
      // unindent
      lines[index] = lines[index].slice(4)
}
lines.splice(index, 1)
```

Reply:
d9e5ad7ee5070bdfd36801254d51a7bfe72baa20